modules/ROOT/pages/single-sign-on-with-saml.adoc

```diff
@@ -273,7 +273,7 @@ More configuration options can be found in https://www.keycloak.org/docs/latest/
 ----
 
 If your Identity Provider is corectly configured (see the section _Configure the Identity Provider_), you are done.
```

Comment:
If your Identity Provider is correctly configured (see the section Configure the Identity Provider), you are done. => Once your Identity Provider is correctly configured (see the section Configure the Identity Provider), try to access any Bonita Application page, an app page or a form URL (or just http://<host>:<port>/bonita[?tenant=<tenantId>]) and make sure that you are redirected to your Identity Provider to log in (unless you are already logged in). +

Reply:
mix between the 2 phrases proposed.

Reply:
Ok, done